test/relativistic/vector/relativistic_velocity_parallel_to_movement_test.py

```diff
@@ -0,0 +1,72 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from sympy.physics.units import speed_of_light
+from symplyphysics import (
+    assert_equal,
+    units,
+    Quantity,
+    QuantityVector,
+    errors,
+)
+from symplyphysics.laws.relativistic.vector import relativistic_velocity_parallel_to_movement as law
+
+# Description
+## A body is moving parallel to a frame of reference that is moving relative to the laboratory frame of reference.
```

Comment:
I think we decided that body is attached to a proper frame. So body is moving in the frame of reference.